packages/client3/tailwind.config.js

```diff
@@ -33,10 +33,12 @@ module.exports = {
           },
           2: {
             DEFAULT: "#7BF8AF",
-            shade: "#BFED47",
+            shade: "#77EAA7",
+            opacity: "#7BF8AF33",
```

Comment:
make sure that storybook colors are still working with this. Pretty sure this breaks

Reply:
Will do.

Reply:
Works
![Screen Shot 2022-10-04 at 1 13 59 PM](https://user-images.githubusercontent.com/98773781/193883706-e0951b8a-5c34-481b-874d-bea3d2f533d4.png)